react/training/src/redux/game/actions/action.js

```diff
@@ -0,0 +1,9 @@
+export const handleClick = i => ({
```

Comment:
This file should be `/src/redux/game/actions.js` plz check the style guide provided: https://github.com/Wolox/tech-guides/blob/master/react/style-guide.md